src/AmsterdamPHP/Bundle/MeetupBundle/Command/CreateJoindinPlaceholderCommand.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace AmsterdamPHP\Bundle\MeetupBundle\Command;
+
+use CL\Slack\Payload\ChatPostMessagePayload;
+use Codeliner\ArrayReader\ArrayReader;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CreateJoindinPlaceholderCommand
+ */
+class CreateJoindinPlaceholderCommand extends ContainerAwareCommand
+{
+    /**
+     * @var \AmsterdamPHP\Bundle\MeetupBundle\Service\EventService
+     */
+    protected $meetup;
```

Comment:
that's my own style, i don't lock down anything unless there is clear need for it, so i default to protected.